<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>402</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTextBrowser" name="textBrowser"/>
               </item>
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QFontComboBox" name="fontComboBox_FontStyle"/>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_FontSize">
                    <item>
                     <property name="text">
                      <string>5</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>6</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>7</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>8</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>9</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>10</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>11</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>12</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_B">
                    <property name="toolTip">
                     <string>加粗</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_I">
                    <property name="toolTip">
                     <string>倾斜</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_U">
                    <property name="toolTip">
                     <string>下划线</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_Color">
                    <property name="toolTip">
                     <string>颜色</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_Save">
                    <property name="toolTip">
                     <string>保存</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="toolButton_Clean">
                    <property name="toolTip">
                     <string>清空</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="res.qrc">
                      <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_Send">
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_onlineNumber">
              <property name="text">
               <string>在线人数：0人</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_exit">
              <property name="text">
               <string>退出</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>在线列表</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
